{"uid":"78eff82809c29662","name":"test_co_list_agent_office_phone_05_144","fullName":"tests.test_checklist.TestPromotionChecklist#test_co_list_agent_office_phone_05_144","historyId":"b71df4eba58c90f9350d5b004dbf2ea0","time":{"start":1739732259280,"stop":1739732412922,"duration":153642},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_TestPromotionChecklist","time":{"start":1739732022652,"stop":1739732022652,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"passed","steps":[{"name":"Click authorisation button","time":{"start":1739732260022,"stop":1739732260575,"duration":553},"status":"passed","steps":[{"name":"Get authorisation button","time":{"start":1739732260022,"stop":1739732260033,"duration":11},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Source select","time":{"start":1739732261192,"stop":1739732261312,"duration":120},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"source","value":"'144'"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739732262689,"stop":1739732262784,"duration":95},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"1"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739732262784,"stop":1739732270194,"duration":7410},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"\nMetadata = PROPERTY Ok âœ…\n","time":{"start":1739732270217,"stop":1739732270217,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739732270239,"stop":1739732270365,"duration":126},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"2"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739732270365,"stop":1739732280395,"duration":10030},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x561a6f95709a <unknown>\n#1 0x561a6f41e8b0 <unknown>\n#2 0x561a6f46fb99 <unknown>\n#3 0x561a6f46fd21 <unknown>\n#4 0x561a6f4be854 <unknown>\n#5 0x561a6f4959ad <unknown>\n#6 0x561a6f4bbb8b <unknown>\n#7 0x561a6f495753 <unknown>\n#8 0x561a6f46138e <unknown>\n#9 0x561a6f462b51 <unknown>\n#10 0x561a6f92076b <unknown>\n#11 0x561a6f9246f2 <unknown>\n#12 0x561a6f90c8fc <unknown>\n#13 0x561a6f9252e4 <unknown>\n#14 0x561a6f8f09cf <unknown>\n#15 0x561a6f945cd8 <unknown>\n#16 0x561a6f945eb6 <unknown>\n#17 0x561a6f955f16 <unknown>\n#18 0x7f3f80e9caa4 <unknown>\n#19 0x7f3f80f29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class CUSTOMPROPERTY is not mapped","time":{"start":1739732280396,"stop":1739732280396,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739732280417,"stop":1739732280542,"duration":125},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"3"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739732280542,"stop":1739732290586,"duration":10044},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x561a6f95709a <unknown>\n#1 0x561a6f41e8b0 <unknown>\n#2 0x561a6f46fb99 <unknown>\n#3 0x561a6f46fd21 <unknown>\n#4 0x561a6f4be854 <unknown>\n#5 0x561a6f4959ad <unknown>\n#6 0x561a6f4bbb8b <unknown>\n#7 0x561a6f495753 <unknown>\n#8 0x561a6f46138e <unknown>\n#9 0x561a6f462b51 <unknown>\n#10 0x561a6f92076b <unknown>\n#11 0x561a6f9246f2 <unknown>\n#12 0x561a6f90c8fc <unknown>\n#13 0x561a6f9252e4 <unknown>\n#14 0x561a6f8f09cf <unknown>\n#15 0x561a6f945cd8 <unknown>\n#16 0x561a6f945eb6 <unknown>\n#17 0x561a6f955f16 <unknown>\n#18 0x7f3f80e9caa4 <unknown>\n#19 0x7f3f80f29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class DATASYSTEM is not mapped","time":{"start":1739732290587,"stop":1739732290587,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739732290609,"stop":1739732290724,"duration":115},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"4"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739732290724,"stop":1739732300744,"duration":10020},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x561a6f95709a <unknown>\n#1 0x561a6f41e8b0 <unknown>\n#2 0x561a6f46fb99 <unknown>\n#3 0x561a6f46fd21 <unknown>\n#4 0x561a6f4be854 <unknown>\n#5 0x561a6f4959ad <unknown>\n#6 0x561a6f4bbb8b <unknown>\n#7 0x561a6f495753 <unknown>\n#8 0x561a6f46138e <unknown>\n#9 0x561a6f462b51 <unknown>\n#10 0x561a6f92076b <unknown>\n#11 0x561a6f9246f2 <unknown>\n#12 0x561a6f90c8fc <unknown>\n#13 0x561a6f9252e4 <unknown>\n#14 0x561a6f8f09cf <unknown>\n#15 0x561a6f945cd8 <unknown>\n#16 0x561a6f945eb6 <unknown>\n#17 0x561a6f955f16 <unknown>\n#18 0x7f3f80e9caa4 <unknown>\n#19 0x7f3f80f29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class ENUMERATION is not mapped","time":{"start":1739732300745,"stop":1739732300745,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739732300768,"stop":1739732300888,"duration":120},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"5"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739732300888,"stop":1739732310913,"duration":10025},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x561a6f95709a <unknown>\n#1 0x561a6f41e8b0 <unknown>\n#2 0x561a6f46fb99 <unknown>\n#3 0x561a6f46fd21 <unknown>\n#4 0x561a6f4be854 <unknown>\n#5 0x561a6f4959ad <unknown>\n#6 0x561a6f4bbb8b <unknown>\n#7 0x561a6f495753 <unknown>\n#8 0x561a6f46138e <unknown>\n#9 0x561a6f462b51 <unknown>\n#10 0x561a6f92076b <unknown>\n#11 0x561a6f9246f2 <unknown>\n#12 0x561a6f90c8fc <unknown>\n#13 0x561a6f9252e4 <unknown>\n#14 0x561a6f8f09cf <unknown>\n#15 0x561a6f945cd8 <unknown>\n#16 0x561a6f945eb6 <unknown>\n#17 0x561a6f955f16 <unknown>\n#18 0x7f3f80e9caa4 <unknown>\n#19 0x7f3f80f29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PROPERTYROOMS is not mapped","time":{"start":1739732310914,"stop":1739732310914,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739732310937,"stop":1739732311067,"duration":130},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"6"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739732311068,"stop":1739732321112,"duration":10044},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x561a6f95709a <unknown>\n#1 0x561a6f41e8b0 <unknown>\n#2 0x561a6f46fb99 <unknown>\n#3 0x561a6f46fd21 <unknown>\n#4 0x561a6f4be854 <unknown>\n#5 0x561a6f4959ad <unknown>\n#6 0x561a6f4bbb8b <unknown>\n#7 0x561a6f495753 <unknown>\n#8 0x561a6f46138e <unknown>\n#9 0x561a6f462b51 <unknown>\n#10 0x561a6f92076b <unknown>\n#11 0x561a6f9246f2 <unknown>\n#12 0x561a6f90c8fc <unknown>\n#13 0x561a6f9252e4 <unknown>\n#14 0x561a6f8f09cf <unknown>\n#15 0x561a6f945cd8 <unknown>\n#16 0x561a6f945eb6 <unknown>\n#17 0x561a6f955f16 <unknown>\n#18 0x7f3f80e9caa4 <unknown>\n#19 0x7f3f80f29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PROPERTYUNITTYPES is not mapped","time":{"start":1739732321113,"stop":1739732321113,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739732321135,"stop":1739732321260,"duration":125},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"7"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739732321260,"stop":1739732331286,"duration":10026},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x561a6f95709a <unknown>\n#1 0x561a6f41e8b0 <unknown>\n#2 0x561a6f46fb99 <unknown>\n#3 0x561a6f46fd21 <unknown>\n#4 0x561a6f4be854 <unknown>\n#5 0x561a6f4959ad <unknown>\n#6 0x561a6f4bbb8b <unknown>\n#7 0x561a6f495753 <unknown>\n#8 0x561a6f46138e <unknown>\n#9 0x561a6f462b51 <unknown>\n#10 0x561a6f92076b <unknown>\n#11 0x561a6f9246f2 <unknown>\n#12 0x561a6f90c8fc <unknown>\n#13 0x561a6f9252e4 <unknown>\n#14 0x561a6f8f09cf <unknown>\n#15 0x561a6f945cd8 <unknown>\n#16 0x561a6f945eb6 <unknown>\n#17 0x561a6f955f16 <unknown>\n#18 0x7f3f80e9caa4 <unknown>\n#19 0x7f3f80f29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYBUILDING is not mapped","time":{"start":1739732331287,"stop":1739732331287,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739732331308,"stop":1739732331436,"duration":128},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"8"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739732331436,"stop":1739732341469,"duration":10033},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x561a6f95709a <unknown>\n#1 0x561a6f41e8b0 <unknown>\n#2 0x561a6f46fb99 <unknown>\n#3 0x561a6f46fd21 <unknown>\n#4 0x561a6f4be854 <unknown>\n#5 0x561a6f4959ad <unknown>\n#6 0x561a6f4bbb8b <unknown>\n#7 0x561a6f495753 <unknown>\n#8 0x561a6f46138e <unknown>\n#9 0x561a6f462b51 <unknown>\n#10 0x561a6f92076b <unknown>\n#11 0x561a6f9246f2 <unknown>\n#12 0x561a6f90c8fc <unknown>\n#13 0x561a6f9252e4 <unknown>\n#14 0x561a6f8f09cf <unknown>\n#15 0x561a6f945cd8 <unknown>\n#16 0x561a6f945eb6 <unknown>\n#17 0x561a6f955f16 <unknown>\n#18 0x7f3f80e9caa4 <unknown>\n#19 0x7f3f80f29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYLASTMARKETSALE is not mapped","time":{"start":1739732341470,"stop":1739732341470,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739732341490,"stop":1739732341621,"duration":131},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"9"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739732341621,"stop":1739732351642,"duration":10021},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x561a6f95709a <unknown>\n#1 0x561a6f41e8b0 <unknown>\n#2 0x561a6f46fb99 <unknown>\n#3 0x561a6f46fd21 <unknown>\n#4 0x561a6f4be854 <unknown>\n#5 0x561a6f4959ad <unknown>\n#6 0x561a6f4bbb8b <unknown>\n#7 0x561a6f495753 <unknown>\n#8 0x561a6f46138e <unknown>\n#9 0x561a6f462b51 <unknown>\n#10 0x561a6f92076b <unknown>\n#11 0x561a6f9246f2 <unknown>\n#12 0x561a6f90c8fc <unknown>\n#13 0x561a6f9252e4 <unknown>\n#14 0x561a6f8f09cf <unknown>\n#15 0x561a6f945cd8 <unknown>\n#16 0x561a6f945eb6 <unknown>\n#17 0x561a6f955f16 <unknown>\n#18 0x7f3f80e9caa4 <unknown>\n#19 0x7f3f80f29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYLOCATION is not mapped","time":{"start":1739732351643,"stop":1739732351643,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739732351664,"stop":1739732351796,"duration":132},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"10"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739732351796,"stop":1739732361858,"duration":10062},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x561a6f95709a <unknown>\n#1 0x561a6f41e8b0 <unknown>\n#2 0x561a6f46fb99 <unknown>\n#3 0x561a6f46fd21 <unknown>\n#4 0x561a6f4be854 <unknown>\n#5 0x561a6f4959ad <unknown>\n#6 0x561a6f4bbb8b <unknown>\n#7 0x561a6f495753 <unknown>\n#8 0x561a6f46138e <unknown>\n#9 0x561a6f462b51 <unknown>\n#10 0x561a6f92076b <unknown>\n#11 0x561a6f9246f2 <unknown>\n#12 0x561a6f90c8fc <unknown>\n#13 0x561a6f9252e4 <unknown>\n#14 0x561a6f8f09cf <unknown>\n#15 0x561a6f945cd8 <unknown>\n#16 0x561a6f945eb6 <unknown>\n#17 0x561a6f955f16 <unknown>\n#18 0x7f3f80e9caa4 <unknown>\n#19 0x7f3f80f29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYOWNERSHIP is not mapped","time":{"start":1739732361859,"stop":1739732361859,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739732361892,"stop":1739732362025,"duration":133},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"11"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739732362025,"stop":1739732372036,"duration":10011},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x561a6f95709a <unknown>\n#1 0x561a6f41e8b0 <unknown>\n#2 0x561a6f46fb99 <unknown>\n#3 0x561a6f46fd21 <unknown>\n#4 0x561a6f4be854 <unknown>\n#5 0x561a6f4959ad <unknown>\n#6 0x561a6f4bbb8b <unknown>\n#7 0x561a6f495753 <unknown>\n#8 0x561a6f46138e <unknown>\n#9 0x561a6f462b51 <unknown>\n#10 0x561a6f92076b <unknown>\n#11 0x561a6f9246f2 <unknown>\n#12 0x561a6f90c8fc <unknown>\n#13 0x561a6f9252e4 <unknown>\n#14 0x561a6f8f09cf <unknown>\n#15 0x561a6f945cd8 <unknown>\n#16 0x561a6f945eb6 <unknown>\n#17 0x561a6f955f16 <unknown>\n#18 0x7f3f80e9caa4 <unknown>\n#19 0x7f3f80f29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYOWNERSHIPTRANSFER is not mapped","time":{"start":1739732372037,"stop":1739732372037,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739732372056,"stop":1739732372204,"duration":148},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"12"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739732372204,"stop":1739732382222,"duration":10018},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x561a6f95709a <unknown>\n#1 0x561a6f41e8b0 <unknown>\n#2 0x561a6f46fb99 <unknown>\n#3 0x561a6f46fd21 <unknown>\n#4 0x561a6f4be854 <unknown>\n#5 0x561a6f4959ad <unknown>\n#6 0x561a6f4bbb8b <unknown>\n#7 0x561a6f495753 <unknown>\n#8 0x561a6f46138e <unknown>\n#9 0x561a6f462b51 <unknown>\n#10 0x561a6f92076b <unknown>\n#11 0x561a6f9246f2 <unknown>\n#12 0x561a6f90c8fc <unknown>\n#13 0x561a6f9252e4 <unknown>\n#14 0x561a6f8f09cf <unknown>\n#15 0x561a6f945cd8 <unknown>\n#16 0x561a6f945eb6 <unknown>\n#17 0x561a6f955f16 <unknown>\n#18 0x7f3f80e9caa4 <unknown>\n#19 0x7f3f80f29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYPRIORSALE is not mapped","time":{"start":1739732382223,"stop":1739732382223,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739732382243,"stop":1739732382399,"duration":156},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"13"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739732382399,"stop":1739732392444,"duration":10045},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x561a6f95709a <unknown>\n#1 0x561a6f41e8b0 <unknown>\n#2 0x561a6f46fb99 <unknown>\n#3 0x561a6f46fd21 <unknown>\n#4 0x561a6f4be854 <unknown>\n#5 0x561a6f4959ad <unknown>\n#6 0x561a6f4bbb8b <unknown>\n#7 0x561a6f495753 <unknown>\n#8 0x561a6f46138e <unknown>\n#9 0x561a6f462b51 <unknown>\n#10 0x561a6f92076b <unknown>\n#11 0x561a6f9246f2 <unknown>\n#12 0x561a6f90c8fc <unknown>\n#13 0x561a6f9252e4 <unknown>\n#14 0x561a6f8f09cf <unknown>\n#15 0x561a6f945cd8 <unknown>\n#16 0x561a6f945eb6 <unknown>\n#17 0x561a6f955f16 <unknown>\n#18 0x7f3f80e9caa4 <unknown>\n#19 0x7f3f80f29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYSEARCH is not mapped","time":{"start":1739732392445,"stop":1739732392445,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739732392465,"stop":1739732392619,"duration":154},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"14"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739732392619,"stop":1739732402662,"duration":10043},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x561a6f95709a <unknown>\n#1 0x561a6f41e8b0 <unknown>\n#2 0x561a6f46fb99 <unknown>\n#3 0x561a6f46fd21 <unknown>\n#4 0x561a6f4be854 <unknown>\n#5 0x561a6f4959ad <unknown>\n#6 0x561a6f4bbb8b <unknown>\n#7 0x561a6f495753 <unknown>\n#8 0x561a6f46138e <unknown>\n#9 0x561a6f462b51 <unknown>\n#10 0x561a6f92076b <unknown>\n#11 0x561a6f9246f2 <unknown>\n#12 0x561a6f90c8fc <unknown>\n#13 0x561a6f9252e4 <unknown>\n#14 0x561a6f8f09cf <unknown>\n#15 0x561a6f945cd8 <unknown>\n#16 0x561a6f945eb6 <unknown>\n#17 0x561a6f955f16 <unknown>\n#18 0x7f3f80e9caa4 <unknown>\n#19 0x7f3f80f29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYSITE is not mapped","time":{"start":1739732402663,"stop":1739732402663,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739732402683,"stop":1739732402838,"duration":155},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"15"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739732402838,"stop":1739732412863,"duration":10025},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x561a6f95709a <unknown>\n#1 0x561a6f41e8b0 <unknown>\n#2 0x561a6f46fb99 <unknown>\n#3 0x561a6f46fd21 <unknown>\n#4 0x561a6f4be854 <unknown>\n#5 0x561a6f4959ad <unknown>\n#6 0x561a6f4bbb8b <unknown>\n#7 0x561a6f495753 <unknown>\n#8 0x561a6f46138e <unknown>\n#9 0x561a6f462b51 <unknown>\n#10 0x561a6f92076b <unknown>\n#11 0x561a6f9246f2 <unknown>\n#12 0x561a6f90c8fc <unknown>\n#13 0x561a6f9252e4 <unknown>\n#14 0x561a6f8f09cf <unknown>\n#15 0x561a6f945cd8 <unknown>\n#16 0x561a6f945eb6 <unknown>\n#17 0x561a6f955f16 <unknown>\n#18 0x7f3f80e9caa4 <unknown>\n#19 0x7f3f80f29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYTAXASSESSMENT is not mapped","time":{"start":1739732412864,"stop":1739732412864,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":48,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"_unittest_setUpClass_fixture_TestPromotionChecklist::0","time":{"start":1739748083629,"stop":1739748083629,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"@pytest.mark.filterwarnings('ignore::DeprecationWarning')"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_checklist"},{"name":"subSuite","value":"TestPromotionChecklist"},{"name":"host","value":"fv-az1718-879"},{"name":"thread","value":"3355-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_checklist"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["@pytest.mark.filterwarnings('ignore::DeprecationWarning')"]},"source":"78eff82809c29662.json","parameterValues":[]}